roles/v2/pool/pool-config.toml

```diff
@@ -1,6 +1,6 @@
 listen_address = "0.0.0.0:34254"
-# tp_address = "127.0.0.1:8442"
-tp_address = "75.119.150.111:8442"
+tp_address = "127.0.0.1:8442"
+# tp_address = "75.119.150.111:8442"
```

Comment:
Let's keep this pointed at the hosted one - the INTEROPERABILITY.md says that the pool connects directly to the hosted TP.